src/crypto/openssl/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+## openssl
+
+set(openssl_crypto_plugin_srcs
+  openssl_crypto_accel.cc
+  openssl_crypto_plugin.cc)
+
+add_library(ceph_crypto_openssl SHARED ${openssl_crypto_plugin_srcs})
+target_link_libraries(ceph_crypto_openssl PUBLIC crypto)
+add_dependencies(crypto_plugins ceph_crypto_openssl)
+set_target_properties(ceph_crypto_openssl PROPERTIES VERSION 1.0.0 SOVERSION 1)
```

Comment:
no need to version the plugin using the soversion. it's already versioned using `__ceph_plugin_version()`.